tests/blackbox/stratis_cli_cert.py

```diff
@@ -180,6 +181,47 @@ def test_filesystem_list_empty(self):
         """
         self.unittest_command([STRATIS_CLI, "filesystem", "list"], 0, True, False)
 
+    def test_key_set_unset(self):
```

Comment:
I think ```stderr``` and ```stdout``` should both be expected to be empty on success on these key management commands.